src/components/FlawAffects/__tests__/FlawAffects.spec.ts

```diff
@@ -1,67 +1,120 @@
+import { type Component } from 'vue';
+
 import { describe, expect } from 'vitest';
+import { createPinia, setActivePinia } from 'pinia';
+import { useRouter } from 'vue-router';
+import { flushPromises, VueWrapper } from '@vue/test-utils';
+
+// import { flushPromises } from '@vue/test-utils';
 
+import { osimFullFlawTest, osimEmptyFlawTest } from '@/components/__tests__/test-suite-helpers';
 import sampleTrackersQueryResult from '@/components/__tests__/__fixtures__/sampleTrackersQueryResult.json';
-import { osimEmptyFlawTest, osimFullFlawTest } from '@/components/__tests__/test-suite-helpers';
 
+import { useFlaw } from '@/composables/useFlaw';
+// import { useFlawModel } from '@/composables/useFlawModel';
+import { useFlawAffectsModel } from '@/composables/useFlawAffectsModel';
+
+// import sampleFlawFull from '@/__tests__/__fixtures__/sampleFlawFull.json';
+import { useAffectsEditingStore } from '@/stores/AffectsEditingStore';
+import { mountWithConfig, withSetup } from '@/__tests__/helpers';
 import { getTrackersForFlaws } from '@/services/TrackerService';
-import { mountWithConfig } from '@/__tests__/helpers';
-import type { ZodFlawType } from '@/types/zodFlaw';
 import { getNextAccessToken } from '@/services/OsidbAuthService';
+import type { ZodFlawType } from '@/types';
 
 vi.mock('@/services/OsidbAuthService');
 vi.mock('@/services/TrackerService');
+vi.mock('@/composables/useFlaw');
+vi.mock('@/composables/useFlawModel');
+vi.mock('@/composables/useFlawAffectsModel');
+vi.mock('@/stores/AffectsEditingStore');
 
-const mountFlawAffects = (flaw: ZodFlawType) => mountWithConfig(FlawAffects, {
-  props: {
-    embargoed: flaw.embargoed,
-    flaw,
-    relatedFlaws: [flaw],
-    error: [],
-  },
-});
+let pinia: ReturnType<typeof createPinia>;
+
+async function useMocks(flaw: ZodFlawType) {
```

Comment:
Is this reusable? May we want to have it in a external file (e.g. test helpers)?